glide.yaml

```diff
@@ -70,3 +70,13 @@ import:
   version: 335da9dda11408a34b64344f82e9c03779b71673
   repo: https://github.com/openshift/glog.git
   vcs: git
+- package: github.com/stretchr/testify
```

Comment:
@hferentschik I couldn't find any good library other than this. Also, it seems to be popular than others.